python/GangaDirac/Lib/Splitters/SplitterUtils.py

```diff
@@ -3,7 +3,7 @@
 from GangaDirac.Lib.Utilities.DiracUtilities import execute
 from GangaDirac.Lib.Backends.DiracUtils import result_ok
 from Ganga.Utility.Config import getConfig
-from GangaDirac.Lib.Files import DiracFile
+from GangaDirac.Lib.Files.DiracFile import DiracFile
```

Comment:
If this changes in this way is it corrected elsewhere in the file, if it doesn't need to be, is it even called?